.idea/misc.xml

```diff
@@ -3,8 +3,11 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
+        <entry key="..\:/habitSmasher/app/src/main/res/layout/fragment_home.xml" value="0.2046875" />
```

Comment:
This looks like a local path judging by the `../:/habitSmasher`!

- [ ] remove local path